pages/index.mdx

```diff
@@ -25,6 +25,7 @@ OpenModelZ forms the core of our [ModelZ](https://modelz.ai) platform, which is
 You can install OpenModelZ using the following command:
 
 ```text copy
+# Installing and using openmodelz this way is currently only supported on Linux.
```

Comment:
We have a doc for macOS actually: https://docs.open.modelz.ai/server/start#start-the-server-on-macos-or-windows

Maybe you could move it to the index.

Reply:
Will do. 

The link won't open, got `Internal Server Error` on my side. Could you please check if the link is correct?

Reply:
The link is correct but it cannot be opened directly... I think it's a bug of nextra. cc @VoVAllen